### PR TITLE
WB-ify ExpressionEditor

### PR DESCRIPTION
**Pull request by handeyeco**
## Summary:
Move ExpressionEditor to using WB.

Issue: Part of [LEMS-2222](https://khanacademy.atlassian.net/browse/LEMS-2222)

## Test plan:
Behavior shouldn't change, just move components over to WB.

[LEMS-2222]: https://khanacademy.atlassian.net/browse/LEMS-2222?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/perseus-editor/src/widgets/expression-editor.tsx [hunk 1]
```diff
@@ -491,58 +493,62 @@ class AnswerOption extends React.Component<
...
-            removeButton = (
-                <button
-                    type="button"
-                    className="simple-button orange"
+        const removeButton = this.state.deleteFocused ? (
+            <>
+                <Button
+                    size="small"
                     onClick={this.handleImSure}
-                    onBlur={this.handleDeleteBlur}
```
Comment (handeyeco) [inline packages/perseus-editor/src/widgets/expression-editor.tsx:510]: WB buttons don't have `onBlur`, so I used a two button system

File: packages/perseus/src/components/text-input.tsx [hunk 2]
```diff
@@ -63,7 +63,6 @@ class TextInput extends React.Component<Props> {
         const formattedValue = value === null ? "" : value.toString();
 
         return (
-            // $FlowIgnore
```
Comment (handeyeco) [inline packages/perseus/src/components/text-input.tsx:66]: We don't use Flow anymore

File: packages/perseus-editor/src/components/sortable.tsx [hunk 3]
```diff
@@ -1,11 +1,17 @@
...
 import PropTypes from "prop-types";
 import * as React from "react";
 import ReactDOM from "react-dom";
 
 const PT = PropTypes;
 
-// Takes an array of components to sort
+/**
+ * Takes an array of components to sort.
+ * As of this writing, there are two sortable components
```
Comment (anakaren-rojas) [inline packages/perseus-editor/src/components/sortable.tsx:11]: nit: `As of 08/05/24` vs `As of this writing` might be helpful for future us

File: packages/perseus-editor/src/widgets/expression-editor.tsx [hunk 4]
```diff
@@ -491,58 +493,62 @@ class AnswerOption extends React.Component<
...
-                <div className="answer-handle" />
+            <div className={css(styles.answerOption)}>
+                <div className={css(styles.answerHandle)} />
 
-                <div className="answer-body">
-                    <div className="answer-considered">
-                        <div
+                <div className={css(styles.answerBody)}>
+                    <div>
+                        <button
```
Comment (jeremywiebe) [inline packages/perseus-editor/src/widgets/expression-editor.tsx:539]: Does it make sense to migrate this to be a WB button right away (or at least a Clickable)?

**Review by jeremywiebe — APPROVED** [review]
Nice to see more wb-ifying going on!  Thanks!

File: packages/perseus-editor/src/widgets/expression-editor.tsx (hunk 4 above)
Comment (handeyeco) [reply packages/perseus-editor/src/widgets/expression-editor.tsx:539]: I tried to move it to Clickable (WB button I think would be too difficult to customize) and I ran into the most bizarre bug I've ever seen and now I'm tired and I don't want to fight it anymore so it's staying a `button`.

https://khanacademy.slack.com/archives/CEA6W0F6F/p1722982631694429

**Pull request merged**